cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -656,19 +648,14 @@ SwiftGenerator::validateMetadata(const UnitPtr& u)
 void
 SwiftGenerator::validateSwiftModuleMappings(const UnitPtr& unit)
 {
-    using ModuleMap = std::map<std::string, std::string>;
-    // Each Slice unit has to map all top-level modules to a single Swift module
-    ModuleMap mappedModules;
-    // With a given Swift module a Slice module has to map to a single prefix
-    std::map<std::string, ModuleMap> mappedModulePrefixes;
+    // Each Slice unit has to map all top-level modules to a single Swift module.
+    map<string, string> mappedModules;
 
     // Any modules that are directly contained on the unit are (by definition) top-level modules.
     // And since there is only one unit per compilation, this must be all the top-level modules.
     for (const auto& module : unit->modules())
```

Comment:
I would not use module as a variable name. It's an identifier with special meaning in C++20: https://en.cppreference.com/w/cpp/keyword

Reply:
Fixed.
I renamed it to `mod`.